src/figuresClasses.ts

```diff
@@ -1,19 +1,101 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle',
+  color: 'red' | 'green' | 'blue',
+  getArea: () => number,
 }
 
-export class Triangle {
+function isTriangle(a: number, b: number, c: number): boolean {
+  if (a > b && a > c && a >= b + c) {
+    return false;
+  }
+
+  if (b > a && b > c && b >= a + c) {
+    return false;
+  }
 
+  if (c > a && c > b && c >= a + b) {
+    return false;
+  }
```

Comment:
better use separate variable for each condition and then
if `var1 || var2 || var3` return false

Reply:
or you can make array with sides, sort it, and use only one check